libp2p/protocols/identify.nim

```diff
@@ -9,8 +9,8 @@
 
 {.push raises: [Defect].}
 
-import options
-import chronos, chronicles
+import std/[sequtils, strutils]
+import chronos, chronicles, json_serialization/std/options
```

Comment:
Hmm, this seems weird, _do we have to_ import this explicitly or is this a workaround?

Reply:
The default option will be serialized into: `Some("text")` (and the " will break json) 
This imports contains a wrapper which will correctly serialize it.

Other option is how I had to do for the observed address

Reply:
Not happy with it either, since it's the first time we mention json_serialization in libp2p (it's a transient dependency)
Oh well, I'll probably switch to the manual technique next week